frontend/styles/_print.scss

```diff
@@ -22,4 +22,10 @@
   .page-break-after-avoid {
     page-break-after: avoid;
   }
+
+  .c-tabs {
```

Comment:
I know I told you to put the styles here, but actually it made more sense inside the profile-actor and profile-place layouts, we are adding page specific print styles there. I made the change myself.